code/game/gamemodes/clock_cult/__clock_defines.dm

```diff
@@ -0,0 +1,15 @@
+var/global/clockwork_construction_value = 0 //The total value of all structures built by the clockwork cult
```

Comment:
It's either for static object vars or explicitly saying you're accessing a global like this: `global.machines`